### ISSUE TITLE
LR scheduler issue with multiple parameter groups

### ISSUE THREAD

**Issue by usuyama** [microsoft/hi-ml#611]
https://github.com/microsoft/hi-ml/blob/7fe55b0c177c4708fad2846c709ff16b2feac001/hi-ml/src/health_ml/utils/lr_scheduler.py#L43

Need to return LRs for each param group
```
    def get_lr(self) -> List[float]:  # type: ignore
        return [self.final_lr * self.warmup_multiplier() for g in self.optimizer.param_groups]
```

**Comment by ant0nsc** [comment]
Thanks for reporting @usuyama - would you like to have a go at this bug?
